Interface/Harp.SoundCard/SampleType.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Harp.SoundCard
+{
+    /// <summary>
+    /// Specifies the available bit depths for sound waveform samples.
+    /// </summary>
+    internal enum SampleType : int
```

Comment:
Not sure about this name.  Cant we just call it `Depth` to align to the OpenCV.Mat?

Reply:
When that happens we can rename everything, right now there is no support for any other type. I will remove the doc strings just to make it more clear this needs to be revisited before making public.